keps/sig-apps/3939-allow-replacement-when-fully-terminated/README.md

```diff
@@ -800,6 +888,9 @@ feature enablement causes the number of syncs to increase.
 
 #### Are there any missing metrics that would be useful to have to improve observability of this feature?
 
+We identified that a new metric would be necessary to improve observability.  
+We will add `job_pods_creation_total` for this reason.
+
```

Comment:
For scalability section

> Will enabling / using this feature result in any new API calls

Won't we potentially observe more API calls due to updating the `Terminating` field?

Reply:
yes, I suppose we would if the pods weren't being terminated.  Its hard to say if we would observe more API calls though as we bulk update the status in the SyncJob function.  So if a pod was terminating or failed or active pods changed, we would trigger an update.  

So I can't really say if we would observe more API calls.